db/schema.rb

```diff
@@ -33,7 +33,7 @@
   end
 
   create_table 'notes', force: :cascade do |t|
-    t.string 'gravity_user_id'
+    t.string 'gravity_user_id', null: false
```

Comment:
@erikdstock do you know what's up here? I don't see a place where we added a migration to mark this is null false so maybe it was a mistake?

Reply:
It was a mistake in #704 - I made it null false there but forgot to re-run the migration to regenerate the schema.